terraform/modules/data_factory/ingestion-pipeline.json

```diff
@@ -110,19 +103,26 @@
                 },
                 "userProperties": [],
                 "typeProperties": {
-                    "url": "${ingestion_container_url}/fhir/linkage/link/add_patient_identifier_in_bundle",
+                    "url": "${ingestion_container_url}/fhir/harmonization/standardization/standardize_phones",
                     "method": "POST",
                     "body": {
-                        "value": "{\n    \"bundle\":@{activity('geocode').output.bundle},\n    \"salt_str\" : \"some-dummy-salt\"\n}    ",
+                        "value": "{\n    \"data\":@{activity('standardize_name').output.bundle}\n}",
```

Comment:
This is completely my fault, because I suggested taking this approach to parameterizing the salt, but I believe this approach will expose the value of the salt to users in Azure. I think we should instead set the salt as an environment variable on ingestion container app.